PluginExampleProject/src/main/scala/com/alpine/plugin/samples/ver1_0/ColumnFilterPlugin.scala

```diff
@@ -0,0 +1,195 @@
+package com.alpine.plugin.samples.ver1_0
+
+import com.alpine.plugin.core._
+import com.alpine.plugin.core.datasource.OperatorDataSourceManager
+import com.alpine.plugin.core.dialog.{ColumnFilter, OperatorDialog}
+import com.alpine.plugin.core.io._
+import com.alpine.plugin.core.spark.utils.SparkUtils
+import com.alpine.plugin.core.spark.{SparkIOTypedPluginJob, SparkRuntimeWithIOTypedJob}
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.spark.SparkContext
+
+import scala.collection.mutable
+
+class ColumnFilterPlugin {
+
+}
+
+class ColumnFilterSignature extends OperatorSignature[
+  ColumnFilterGUINode,
+  ColumnFilterRuntime] {
+  def getMetadata(): OperatorMetadata = {
+    new OperatorMetadata(
+      name = "ColumnFilter (Spark)",
+      category = "Transformation",
+      author = "Egor Pakhomov",
+      version = 1,
+      helpURL = "",
+      iconNamePrefix = ""
+    )
+  }
+}
+
+
+class ColumnFilterGUINode extends
+OperatorGUINode[HdfsTabularDataset, HdfsTabularDataset] {
+  override def onPlacement(
+                            operatorDialog: OperatorDialog,
+                            operatorDataSourceManager: OperatorDataSourceManager,
+                            operatorSchemaManager: OperatorSchemaManager): Unit = {
+
+    operatorDialog.addTabularDatasetColumnCheckboxes(
+      "columnsToDelete",
+      "Columns to delete",
+      ColumnFilter.All,
+      "main"
+    )
+
+    operatorDialog.addDropdownBox(
+      "storageFormat",
+      "Storage format",
+      Array("Parquet", "Avro", "Tsv").toSeq,
+      "Parquet"
+    )
+
+    operatorDialog.addHdfsFileSelector(
```

Comment:
OutputParameters also need to be updated. See https://github.com/alpinedatalabs/adl/commit/62490c66b56b54e1ce3ed5bd1ae76043fd8c29ad
(this isn't published yet, so it might be easier for you to check this stuff into adl, and we'll move everything over when we publish again).